src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -64,6 +66,17 @@ class OrchestratorCli(MgrModule):
             "cmd": "orchestrator status",
             "desc": "Report configured backend and its status",
             "perm": "r"
+        },
+        {   'cmd': "orchestrator osd create "
```

Comment:
Would it be possible to split changes of the `orchestrator_cli` module and the `orchestrator.py` in a separate PR? This is not required, but it would keep the size of this PR a bit smaller

Reply:
I suggest to "rebase" this PR once your modifications in #25912 will be merged. I hope next PR's will be smaller.